app/models/user.rb

```diff
@@ -1,6 +1,10 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  after_create :make_admin
+
```

Comment:
- [ ] Using the `after_create` callback to call the `make_admin` method will make all `users` that are created an _**admin**_ and give them **_admin privileges_**. In a real-world situation, you don't want all the users to have admin privileges because it can be disastrous 😄 . Kindly remove this line of code.